{{cookiecutter.project_name}}/tests/generated_project_tests/test_entry_points.py

```diff
@@ -0,0 +1,46 @@
+import pytest
```

Comment:
Is this a duplicate of `tests/project_tests/test_entry_points.py`? If so, is there a way to have the file only once?

Reply:
Yes, this is a duplicate file of the original `test_entry_points.py` in the project, it ensures that the entry points are working well within a `generated project` using a template. We had a discussion previously to move most of the things out of the project into the template except the `project generation tests` and `publishing CI`. In the long run, we wouldn't be maintaining or testing the project much, rather, we would be testing the template, i.e. generating a project using various kinds of build backends and such and testing the units inside them, because at the end of the day, we are going to deliver the template to the users, so testing the project doesn't make much sense. So we would be removing the copy of `test_entry_points.py` from the project and keeping it in the template, if that makes sense.

Also in the previous meeting, we discussed about providing a CLI with the template, so that a user doesn't have to go through a hassle to figure things out, just one command in the command line would set up a project in the current working directory. This would also make it easy to publish things to PyPI, manage releases, and support versions of PyBaMM(because some of the code within the template is derived from PyBaMM so we might need to update things as things get updated upstream). So to sum it up, 
```bash
pip install pybamm-cookiecutter
pybamm-cookiecutter
```
would basically initialise an empty project in the current working directory of your terminal.